### PR TITLE
Fix discrepancy between link and link text

### PR DESCRIPTION
**Pull request by osg-grafana**

### CODE REVIEW

**Review by brendamuir — APPROVED** [review]

File: docs/static/templates/reference-template.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ Add an introduction to the reference.
 
 <!-- The introduction is required. Include reference information, such as extensive tables, lists, or other information that is used as support for a task. Reference topics are also designed for API information.
 
-Often reference topics are linked from _task_ topics, because they contain information the user needs in order to perform a task. 
+For guidelines about writing a reference topic, see [Reference topic](https://grafana.com/docs/writers-toolkit/writing-guide/topic-types/reference/).
```
Comment (chri2547) [inline docs/static/templates/reference-template.md:23]: Should this be a relref?

**Review by chri2547 — APPROVED** [review]
A question for you. Thank you for the contribution!

File: docs/static/templates/reference-template.md (hunk 1 above)
Comment (osg-grafana) [reply docs/static/templates/reference-template.md]: Not sure, due to static content. Will test and see what happens.

Comment (osg-grafana) [reply docs/static/templates/reference-template.md:23]: No, because it is within a comment and does not get rendered.

**Pull request merged**